apps/admin/src/app/tracks/[trackId]/_components/update-track-form.tsx

```diff
@@ -175,9 +175,39 @@ export function UpdateTrackForm({ challenges, track }: Props) {
             <Select
               isMulti
               styles={{
-                option: () => {
+                option: (base, { data, isDisabled, isFocused, isSelected }) => {
+                  console.log(base.backgroundColor);
                   return {
```

Comment:
whats all this stuff do.

Reply:
This is what you have to do to overwrite the base style. For now this works but I think this should be a custom radixui comp to make it align with the rest of our ui stuff